tests/test_fock.py

```diff
@@ -629,6 +645,23 @@ def circuit(n, a):
         expected = np.array([2 * a ** 2 + 2 * n + 1, 2 * a * (2 * n + 1)])
         assert np.allclose(gradF, expected, atol=tol, rtol=0)
 
+    def test_polyxp_variance(self, tol):
+        """Tests that variance for PolyXP measurement works"""
+        dev = qml.device("strawberryfields.fock", wires=1, cutoff_dim=15)
+
+        @qml.qnode(dev)
+        def circuit(r, phi):
+            qml.Squeezing(r, 0, wires=0)
+            qml.Rotation(phi, wires=0)
+            return qml.var(qml.PolyXP(np.array([0, 1, 0]), wires=0))
```

Comment:
Idea here is to have `PolyXP` equal to `X` and recycle the expected data from previous tests.

Reply:
Would it be worth adding cases for unique wire labels?